.pre-commit-config.yaml

```diff
@@ -1,11 +1,6 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
-- repo: https://github.com/dragomirecky/cmake-format-pre-commit-hook
```

Comment:
I don't agree with removal of cmake formatter

Reply:
IMHO, It's time to specify what formatting we don't want instead of how we see the best style.